README.md

```diff
@@ -157,6 +157,7 @@ See also [Static Site Generators](#static-site-generators) and  [Content Managem
   * [Friendica](http://friendica.com/) - Social Communication Server ([Source code](https://github.com/friendica/friendica)) `AGPLv3`
   * [Gnu Social](http://gnu.io/social/) - social communication software for both public and private communications
   * [Hubzilla](https://github.com/redmatrix/hubzilla) - independent general-purpose websites that not only connect with their associated members and viewers, but also connect together `MIT` `PHP`
+  * [HumHub](https://github.com/humhub/humhub) - A flexible kit for private social networks `AGPLv3` `PHP`
```

Comment:
Link should point to https://www.humhub.org/.
Github link can be added as `([Source code](https://github.com/humhub/humhub))`(see other links)